libraries/lib-project-rate/ProjectRate.cpp

```diff
@@ -80,16 +81,15 @@ void ProjectRate::SetRate(double rate)
 
 static ProjectFileIORegistry::WriterEntry entry {
 [](const AudacityProject &project, XMLWriter &xmlFile){
-   xmlFile.WriteAttr(wxT("rate"), ProjectRate::Get(project).GetRate());
+   xmlFile.WriteAttr("rate", ProjectRate::Get(project).GetRate());
```

Comment:
I notice this change to the call to WriteAttr.  (Only to this call.)  But the function XMLWriter::WriteAttr was not changed.  There is still an implicit construction of a wxString each time it is called.

Did you mean yet another change of arguments, to take std::string_view?  This might squeeze some more performance out of serialization.  But this PR is about the deserialization.